hesiod/cfgparse/__init__.py

```diff
@@ -0,0 +1,14 @@
+from typing import Callable, Dict
+
+from hesiod.cfgparse.cfgparser import ConfigParser
+from hesiod.cfgparse.yamlparser import YAMLConfigParser
+
+
+def get_parser(ext: str) -> Callable[..., ConfigParser]:
+    parsers: Dict[str, Callable[..., ConfigParser]] = {"yaml": YAMLConfigParser}
```

Comment:
Each parser should define the supported extensions, so you don't need to repeat `"yaml"` here